patches/chrome-browser-resources-settings-privacy_page-personalization_options.html.patch

```diff
@@ -23,7 +23,7 @@ index 9d35ae4a8be3f92fb0bd665eb08558bea65ba0c6..628d2aabcf87bccbd543c7184a27c2b0
      </template>
  </if><!-- chromeos -->
 +</if><!-- _google_chrome -->
-+<if expr="_chromium">
++<if expr="True">
```

Comment:
below this is a patch for `</if><!-- _google_chrome -->` -> `</if><!-- _chromium -->` which is no longer correct, but also shouldn't be changed at all. Just leave it as the original

Reply:
fixed in 6cdcfa7593a14553422b3c0ed74824033e3dc1e8.